elastic/datadog_checks/elastic/metrics.py

```diff
@@ -302,8 +302,54 @@
         'gauge',
         'indexing_pressure.memory.current.replica_in_bytes',
     ),
+    'elasticsearch.indexing_pressure.memory.current.all_in_bytes': (
+        'gauge',
+        'indexing_pressure.memory.current.all_in_bytes',
+    ),
+    'elasticsearch.indexing_pressure.memory.current.combined_coordinating_and_primary_in_bytes': (
+        'gauge',
+        'indexing_pressure.memory.current.combined_coordinating_and_primary_in_bytes',
+    ),
+    'elasticsearch.indexing_pressure.memory.total.coordinating_in_bytes': (
+        'gauge',
+        'indexing_pressure.memory.total.coordinating_in_bytes',
+    ),
+    'elasticsearch.indexing_pressure.memory.total.primary_in_bytes': (
+        'gauge',
+        'indexing_pressure.memory.total.primary_in_bytes',
+    ),
+    'elasticsearch.indexing_pressure.memory.total.replica_in_bytes': (
+        'gauge',
+        'indexing_pressure.memory.total.replica_in_bytes',
+    ),
+    'elasticsearch.indexing_pressure.memory.total.all_in_bytes': (
+        'gauge',
+        'indexing_pressure.memory.total.all_in_bytes',
+    ),
+    'elasticsearch.indexing_pressure.memory.total.combined_coordinating_and_primary_in_bytes': (
+        'gauge',
+        'indexing_pressure.memory.total.combined_coordinating_and_primary_in_bytes',
+    ),
+    'elasticsearch.indexing_pressure.memory.total.coordinating_rejections': (
+        'gauge',
+        'indexing_pressure.memory.total.coordinating_rejections',
+    ),
+    'elasticsearch.indexing_pressure.memory.total.primary_rejections': (
+        'gauge',
+        'indexing_pressure.memory.total.primary_rejections',
+    ),
+    'elasticsearch.indexing_pressure.memory.total.replica_rejections': (
+        'gauge',
+        'indexing_pressure.memory.total.replica_rejections',
+    ),
 }
 
+ADDITIONAL_METRICS_POST_7_10_0 = {
+    'elasticsearch.indexing_pressure.memory.limit_in_bytes': (
+        'gauge',
+        'indexing_pressure.memory.limit_in_bytes',
+    ),
+}
```

Comment:
Not familiar with this integration, do we have a test to validate these new metrics are correctly collected? 

Also maybe for another PR: add `assert_metrics_using_metadata` to the test to validate the `metadata.csv` file as well

Reply:
I see!